packages/ui/src/SelectField.tsx

```diff
@@ -201,30 +204,58 @@ export const SelectField = <V,>({
     >
       <Label id={id} label={label} className={labelClassName} />
       <div className={styles.selectBlock}>
-        <ReactSelect<SelectFieldOption<V>>
-          inputId={id}
-          className="hz-select-field"
-          classNamePrefix="hz-select-field"
-          // https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-invalid_attribute
-          aria-errormessage={error && errorId(id)}
-          aria-invalid={!!error}
-          aria-required={required}
-          isClearable={isClearable ?? false}
-          isDisabled={disabled}
-          isSearchable={isSearchable}
-          isMulti={isMulti}
-          name={name}
-          value={value}
-          onChange={onChange as (value: ValueType<SelectFieldOption<V>>, action: ActionMeta<SelectFieldOption<V>>) => void}
-          menuPortalTarget={getMenuContainer(menuPortalTarget)}
-          components={{
-            DropdownIndicator,
-            ClearIndicator,
-            Input,
-            MultiValue,
-          }}
-          {...rest}
-        />
+        {isCreatable ? (
+          <ReactSelectCreatable<SelectFieldOption<V>>
```

Comment:
nit: They have the same props, right? Should we then do something like this?
```tsx
const SelectComponent = isCreatable ? ReactSelectCreatable : ReactSelect
...
return <SelectComponent  .../>
```

Reply:
Creatable has the same props as classic select and **some additional** props. I ended up not using any of the extra props, so for now we can change the implementation to look as you suggest.

Reply:
@danad02 alternatively we could create a single object with all props and apply it to both components. It is probably a better approach.

Reply:
```tsx
const props = {}
...
return isCreatable ? <CreatableSelect {...props} /> : <Select {...props} />
```